.pre-commit-config.yaml

```diff
@@ -27,13 +27,13 @@ repos:
     hooks:
       - id: mypy
 
-  - repo: https://github.com/PyCQA/pylint

```

Comment:
why this is commented out?

Reply:
After putting all the code inside src folder, it returns error. And, i am not sure how to fix it